<?xml version="1.0" ?>
<launch>
  <!-- params -->
  <param name="forest_file" value="$(find leg_tracker)/config/trained_leg_detector_res=0.33.yaml" />
  <param name="scan_topic" value="/scan" />
  <param name="fixed_frame" value="map"/>
  <param name="scan_frequency" value="10"/>  
  <param name="dist_travelled_together_to_initiate_leg_pair" value="0.2"/>  
  <param name="cluster_dist_euclid" value="0.05"/>  
  <param name="confidence_threshold_to_maintain_track" value="0.01"/>  
  <param name="max_std" value="0.9999"/>  
  
  <!-- run detect_leg_clusters -->  
  <node pkg="leg_tracker" type="detect_leg_clusters" name="detect_leg_clusters" output="screen"/>

  <!-- run individual_leg_tracker -->
  <node pkg="leg_tracker" type="individual_leg_tracker.py" name="individual_leg_tracker" output="screen"/>
  
</launch>

